include/net/net_ip.h

```diff
@@ -1023,6 +1023,57 @@ static inline bool net_ipv6_is_addr_mcast_group(const struct in6_addr *addr,
 		UNALIGNED_GET(&addr->s6_addr32[3]) == group->s6_addr32[3];
 }
 
+/**
+ * @brief Check if the IPv6 address belongs to the all nodes multicast group
+ *
+ * @param addr IPv6 address
+ *
+ * @return True if the IPv6 multicast address belongs to the all nodes multicast
+ * group, false otherwise
+ */
+static inline bool
+net_ipv6_is_addr_mcast_all_nodes_group(const struct in6_addr *addr)
+{
+	static struct in6_addr all_nodes_mcast_group = {
```

Comment:
We could also make this const, as it never changes.

Reply:
Done.